lib/cisco_node_utils/interface.rb

```diff
@@ -74,6 +74,20 @@ def access_vlan=(vlan)
       raise "[#{@name}] '#{e.command}' : #{e.clierror}"
     end
 
+    def acl
+      config_get('interface', 'acl', @name)
```

Comment:
How could this possibly work? You don't have an `acl` tag in the yaml file?

Please work with Saqib and Shengjun to get this code straightened out. I'll make a few comments here but it needs a total rewrite.